section3/reflection.md

```diff
@@ -1,17 +1,39 @@
+with seeking help. I think it largely stems from not wanting to burden others with
+my struggles (when I have the confidence that I can eventually figure it out myself).
+Also- seek out challenges. This will force you to grow no matter what challenge
+you are facing. It's a great mentality to have.
 
-1. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
+2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
+* I am a constant learner, and do so through reading, and extensive question asking
+(to people im interested in). I try to keep my mind as open as possible, to expose
+myself to as many perspectives as possible.
 
-1. What is a Hash, and how is it different from an Array?
+3. I could definitely do a lot better at goal setting. I really try to focus on the present/here/now, because it seems that many people go their entire lives chasing something that never arrives. I think that's where SMART goals come into play. Goals
+dont have to be 30 year goals, in fact, they are arguably more effective if your
+"TIME" horizon for the goal is short.
 
-1. In the space below, create a Hash stored to a variable named `pet_store`.  This hash should hold an inventory of items and the number of that item that you might find at a pet store.
+4. What is a Hash, and how is it different from an Array?
+* A hash is a way to store data (like an array), but instead of using only numbers to get the data, you can use almost anything. This lets you treat a hash like its a database for storing and organizing data.
 
-1. Given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
+5. In the space below, create a Hash stored to a variable named `pet_store`.  This hash should hold an inventory of items and the number of that item that you might find at a pet store.
+* pet_store = {dogs: 25, cats: 10, fish: 200}
 
-1. With the same hash above, how would we get all the keys?  How about all the values?
+6. Given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
+* puts states.fetch('IA')
 
-1. What is another example of when we might use a hash?  In your example, why is a hash better than an array?
+* Note: puts states.index("Iowa")    would return IA
 
-1. What questions do you still have about hashes?
+7. With the same hash above, how would we get all the keys?  How about all the values?
+* puts states.keys
+* puts states.values
+
+8. What is another example of when we might use a hash?  In your example, why is a hash better than an array?
+* Taking roll call/attendance. Order does not matter, so we would use a hash over an array.
+
+9. What questions do you still have about hashes?
+* No questions, just definitely will want to practice them more prior to the 22nd. Also curious to learn more shortcuts to manipulate the hash.
```

Comment:
You will learn and work with hashes quite often during your turing time! Complex and deeply nested hashes are in your future, but you will be able to handle it for sure